### PR TITLE
Update Tornjak APIs in manager code

### LINKED ISSUE #491: Update Manager to use Tornjak v1 api

### PR DESCRIPTION
**Pull request by maia-iyer**
Closes #491

### CODE REVIEW

File: api/manager/server.go [hunk 1]
```diff
@@ -14,9 +14,9 @@ import (
 	managerdb "github.com/spiffe/tornjak/pkg/manager/db"
 )
 
-var (
+/*var (
```
Comment (mamy-CS) [inline api/manager/server.go:17]: If dead code, remove

**Review by mamy-CS — APPROVED** [review]
Other than the unused variable, /lgtm

File: api/manager/server.go (hunk 1 above)
Comment (maia-iyer) [reply api/manager/server.go]: done

**Review by mamy-CS — APPROVED** [review]
/lgtm

**Pull request merged**